osf-cas/templates/configmap.yaml

```diff
@@ -812,7 +873,7 @@ services/preprints-203948234207240.json: |-
 {{- $domainPrefix := .Values.preprintProviderDomains.prefix -}}
 {{- $domainSuffix := .Values.preprintProviderDomains.suffix -}}
 {{- range $key, $val := (include "cas.jetty.preprint-services" . | fromYaml) }}
-services/{{ $key }}-{{ $val.id }}.json: |-
+services/preprints-{{ $key }}-{{ $val.id }}.json: |-
```

Comment:
Changing this template value is easier than changing the names of all existing files.